diagnostic.md

```diff
@@ -3,53 +3,65 @@
 Using `inspect element`, write four selectors found on the main page for this
 GitHub repo.
 
-<!-- your answer starts here -->
+<!--
+1.  $('#container site-footer-container')
+2.  $('#li')
+3.  $('#h1')
+4.  $('#div id="readme"') -->
 
-<!-- your answer ends here -->
+<!--  -->
 
 ## Question 2
 
 Using the diagram in the `README` file, what are two ways you could return ALL
 list items (`li`) using jQuery? This is intentionally vague. (Assume starting
 at the `OL` DOM node).
 
-<!-- your answer starts here -->
-
-<!-- your answer ends here -->
+<!--$("li").first().next();
+    $("#OL").children("li");
```

Comment:
Close, be careful about capitalization, also `ol` is not an ID so we don't need the `#` before it. Here's one way to do it:

`$('ol > li')`